src/SprykerEco/Yves/FactFinderWebComponents/Model/WebComponentsConfigProvider/WebComponentsConfigProvider.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Model\WebComponentsConfigProvider;
+
+use SprykerEco\Yves\FactFinderWebComponents\FactFinderWebComponentsConfig;
+
+class WebComponentsConfigProvider implements WebComponentsConfigProviderInterface
```

Comment:
Why do we need this class and don't use Config class itself?

Reply:
We use it in factory FactFinderWebComponentsFactory::createFactFinderWebComponentsConfigProvider